logOutputSubject_test.go

```diff
@@ -227,3 +275,69 @@ func TestLogOutputSubject_ClearObservers(t *testing.T) {
 	obs, _ = los.Observers()
 	assert.Equal(t, 0, len(obs))
 }
+
+func TestIsASCII(t *testing.T) {
+	t.Parallel()
+
+	assert.True(t, logger.IsASCII("ascii TEXT 1234 \\~&&\t\n"))
+	assert.False(t, logger.IsASCII("µs"))
+	assert.True(t, logger.IsASCII(testString1))
+	assert.False(t, logger.IsASCII(testString2))
```

Comment:
Perhaps it would be worth here to rename:

- `testString1` -> `testASCIIString1`
- `testString2` -> `testNonASCIIString2`

Reply:
done, removed the trailing "1" and "2"